### DISCUSSION TITLE
Seeing other prompts while using automatic1111 on Gdrive?

### DISCUSSION THREAD

**Discussion by lgooders** [TheLastBen/fast-stable-diffusion#1389] [General]
I was using my own trained model and I started to see other test-img prompts in my "recent" folder on my gdrive.  It was some wild stuff.  I closed my session and changed my google password, then all the images under my "recent" folder disappeared like it was never there.  What would be the reason for that?

**Comment by TheLastBen** [comment]
Use gradio password to avoid that

**Reply by lgooders** [reply]
Bit of a newb......how do I do that LOL

**Reply by TheLastBen** [reply]
in the last cell, there is where to set up a user and a password, make sure you use the latest colab to see that feature

**Reply by lgooders** [reply]
thank you!!!
